packages/ui-extensions/src/surfaces/checkout/components/DropZone/DropZone.doc.ts

```diff
@@ -45,19 +45,19 @@ const data: ReferenceEntityTemplateSchema = {
       sectionContent: `
 ### File storage
 
-File storage for uploads must be implemented separately. Metafields and [the Customer Account API](https://shopify.dev/docs/api/customer/latest/mutations/metafieldsSet) can be utilized to store references to files alongside the relevant objects.
+File storage for uploads must be implemented separately. Metafields and [the Checkout API](https://shopify.dev/docs/api/checkout-ui-extensions/latest/apis/metafields) can be utilized to store references to files alongside the relevant objects.
```

Comment:
Hm I wonder if we should mention both and say to use checkout api for checkout and customer account api when the component is used inside customer accounts?

It's not ideal, but otherwise we'd lose that info entirely